lib/events.js

```diff
@@ -663,7 +663,7 @@ async function _createLocalEventRecord({worker, event, eventHash}) {
   // merge event that merges this event reaches consensus or if the peer
   // becomes a witness on a subsequent block
   const {blockHeight, witnesses} = consensusState;
-  const requiredBlockHeight = witnesses.includes(localPeerId) ?
+  const requiredBlockHeight = witnesses.has(localPeerId) ?
```

Comment:
This LGTM, it makes sense for `witnesses` to be a `Set` inside `consensusState`.